{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入错误的密码", "status": "passed", "attachments": [{"name": "错误的密码", "source": "779ee3ec-0425-4006-b208-01ee3e7291f9-attachment.attach"}], "start": 1651988941670, "stop": 1651988941670}], "attachments": [{"name": "截图", "source": "5c8c8b65-a47b-49b7-b21f-bbb0f4475bae-attachment.png", "type": "image/png"}, {"name": "截图", "source": "d577920a-73f5-4ea0-8ce8-0ce9e459e66c-attachment.png", "type": "image/png"}], "start": 1651988941670, "stop": 1651988943739, "uuid": "931f92d2-3440-4aa0-a873-e12f46153ab9", "historyId": "e042998cd34bf7d92d802f115102d44e", "testCaseId": "ce17e49339328b1be4c6805d1bcb9a3f", "fullName": "test_two.Test#test_22", "labels": [{"name": "severity", "value": "critical"}, {"name": "epic", "value": "xxx测试"}, {"name": "feature", "value": "登录模块"}, {"name": "story", "value": "登录"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "14540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}